packages/itwinui-variables/src/index.scss

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@use './themes/index' as themes;
+@use './space' as *;
+@use './typography' as *;
+@use './shadows' as *;
+@use './colors' as *;
+
+@mixin common {
+  @include sizes;
+  @include border-radii;
+  @include typography;
+  @include shadows;
+  @include soft-backgrounds;
+}
+
+:root {
+  // global vars shared between all themes
+  @include common;
+
+  // default to light theme if data attribute not set anywhere
+  @include themes.light-theme;
```

Comment:
I know I commented before on naming, but looking at this include, maybe enough having just `light`? 
We already know its theme from the file and import

Reply:
In `themes.light-theme`:
- "themes" comes from the import (we say `as themes`)
- "light-" comes from the `@forward` in `themes/index.scss`.
- "theme" comes from the name of the mixin in `themes/light.scss`.

I don't think I can get rid of the `-theme` suffix.